firebase.json

```diff
@@ -48,6 +48,56 @@
           "destination": "https://zksync.io/contact",
           "type": "302"
         },
+        {
+          "source": "/docs/dev/building-on-zksync/hello-world.html",
+          "destination": "/build/quick-start/hello-world.html",
+          "type": "302"
+        },
+        {
+          "source": "/docs/reference/architecture/differences-with-ethereum.html",
+          "destination": "/build/developer-reference/differences-with-ethereum.html",
+          "type": "302"
+        },
+        {
+          "source": "/docs/reference/concepts/hyperscaling.html",
```

Comment:
this is duplicate with the main branch (there are already two for prod and staging)